ui/src/features/components/JobForm/index.js

```diff
@@ -595,7 +613,18 @@ class Form extends React.Component {
         convertedArgs.webhooks = this.state.webhooks.map((webhook) => webhook.key);
       }
 
-      this.props.createJob(createJobRequest(Object.assign({}, this.state, convertedArgs)));
+      const jobData = {
+        ...this.state,
+        ...convertedArgs
+      };
+
+      if (this.props.editMode) {
+        this.setState({ editJob: jobData });
```

Comment:
why do u do that? 
jobData contains also the converted args which is not always suitable to the components as I remember

Reply:
not sure why it was there, it does not affect functionality in any good way